doc/source/usage.rst

```diff
@@ -318,7 +318,8 @@ allows for adding new events. ``khal new`` should understand the following synta
 
 ::
 
-    khal new [-a CALENDAR] [OPTIONS] startdatetime [enddatetime] [timezone] summary [description]
+    khal new [-a CALENDAR] [OPTIONS] [START] [END | DELTA] [TIMEZONE] [SUMMARY]
+    [:: DESCRIPTION]
```

Comment:
I think `khal new [-a CALENDAR] [OPTIONS] [START [END | DELTA] .... summary ... ]` would be clearer, as the order is still important.